performance-metrics/src/main.rs

```diff
@@ -178,9 +178,9 @@ impl PerformanceTest {
     }
 
     // Calculate the timeout for each test
-    // Note: To cover the setup/cleanup time, 20s is added for each iteration of the test
+    // Note: To cover the setup/cleanup time, 40s is added for each iteration of the test
     pub fn calc_timeout(&self) -> u64 {
-        ((self.control.test_timeout + 20) * self.control.test_iterations) as u64
+        ((self.control.test_timeout + 40) * self.control.test_iterations) as u64
```

Comment:
Ditto: by using cold-plugged test image, we won't need to tweak the timeout and wait longer.

Reply:
Yeah